chain/bitcoind_events_test.go

```diff
@@ -338,6 +338,8 @@ func setupBitcoind(t *testing.T, minerAddr string,
 
 	zmqBlockHost := "ipc:///" + tempBitcoindDir + "/blocks.socket"
 	zmqTxHost := "ipc:///" + tempBitcoindDir + "/tx.socket"
+	zmqSequenceHost := "ipc:///" + tempBitcoindDir + "/sequence.socket"
```

Comment:
Will need documentation on the `lnd` side to set. But as mentioned above, I don't think we can actually rely on this unless we start to be stricter with teh bitcoind version required. IIUC, we support bitcoind 22 onwards (post taproot)